CHANGELOG.md

```diff
@@ -3,7 +3,61 @@
+
+### Built-in Functions
+
+- The `net.lookup_ip_addr` function was added to allow policies to resolve hostnames to IPv4/IPv6 addresses ([#3993](https://github.com/open-policy-agent/opa/issues/3993))
+- The `http.send` function has been improved close TCP connections quickly after receiving the HTTP response and avoid creating HTTP clients unnecesarily when a cached response exists ([#4015](https://github.com/open-policy-agent/opa/issues/4015)). This change reduces the number of open file descriptors required in high-throughput environments and prevents OPA from encountering ulimit errors.
+
+### Rego
+
+- `print()` calls in the head of rules no longer cause runtime errors ([#3967](https://github.com/open-policy-agent/opa/issues/3967))
+- Type errors for calls to undefined functions no longer contain rewritten variable names ([#4031](https://github.com/open-policy-agent/opa/issues/4031))
+- The `rego.SkipPartialNamespace` option now correctly sets the flag on the partial evaluation queries (previously it would always set the value to `true`) ([#3996](https://github.com/open-policy-agent/opa/issues/3996)) authored by @[thomascoquet](https://github.com/thomascoquet)
+- The internal set implementation has been updated to insert elements in sorted order rather than lazily sorting during comparisons.
+- Fixed `import` alias parsing bug identified by fuzzer ([#3988](https://github.com/open-policy-agent/opa/issues/3988))
+
+### WebAssembly
+
+- The Golang SDK will now issue a `grow()` call if the `input` document exceeds the available memory space.
+- The `malloc()` implementation will now call `opa_abort` if the `grow()` call fails.
+
+### Server
+
+- The decision logger adapts upload chunk sizes based on previous outputs. This allows the decision loggger to encode significantly more decisions into each upload chunk, thereby reducing heap usage for buffered decisions. For more information on the adapative chunking behaviour, see the [Decision Logs](https://www.openpolicyagent.org/docs/edge/management-decision-logs/) page in the docs.
+- The decision logger can be configured to send records to a custom plugin as well as an HTTP endpoint at the same time ([#4013](https://github.com/open-policy-agent/opa/issues/4013))
+- `print()` calls from the `system.authz` policy are now included in the logs ([#4048](https://github.com/open-policy-agent/opa/issues/4048))
+- OPA can use an [Azure Managed Identities Token](https://www.openpolicyagent.org/docs/edge/configuration/#azure-managed-identities-token) to authenticate with control plane services.
```

Comment:
Missing credits to author here

Reply:
I knew something was wonky with the notes... I think the changelog script is not reporting correctly. I'll look into that...